training/src/main/scala/com/airbnb/aerosolve/training/AdditiveModelTrainer.scala

```diff
@@ -299,17 +311,6 @@ object AdditiveModelTrainer {
     return loss
   }
 
-  private def transformExamples(input: RDD[Example],
-                        config: Config,
-                        key: String,
-                        params: AdditiveTrainerParams): RDD[Example] = {
-    if (params.isRanking) {
-      LinearRankerUtils.transformExamples(input, config, key)
```

Comment:
so we don't do isRanking, what's the difference between transformExamples and makePointwiseFloat, both do combineContextAndItems, but transformExamples unsetContext, what's the role of Context play here?